emap-setup/global-configuration-EXAMPLE.yaml

```
@@ -82,6 +82,13 @@ dates:
 # Configurations for the UDS.
 uds:
   UDS_JDBC_URL: jdbc:postgresql://uds_db:5432/uds
+  # dialect to use for core informdb tables
+  UDS_HIBERNATE_DIALECT: uk.ac.ucl.rits.inform.informdb.PostgreSQLEmapDialect
+  # dialect for modules that don't have access to emap-star (which contains the above code)
+  UDS_HIBERNATE_DIALECT_2: org.hibernate.dialect.PostgreSQL95Dialect

```

Would this be a more meaningful name?
```suggestion
  UDS_HIBERNATE_DIALECT_FALLBACK: org.hibernate.dialect.PostgreSQL95Dialect

```